packages/grafana-data/src/types/teams.ts

```diff
@@ -0,0 +1,58 @@
+export interface Team {
+  id: number; // TODO switch to UUID
+  uid: string; // Prefer UUID
+
+  /**
+   * AccessControl metadata associated with a given resource.
+   */
+  accessControl?: Record<string, boolean>;
+  /**
+   * AvatarUrl is the team's avatar URL.
+   */
+  avatarUrl?: string;
+  /**
+   * Email of the team.
+   */
+  email?: string;
+  /**
+   * MemberCount is the number of the team members.
+   */
+  memberCount: number;
+  /**
+   * Name of the team.
+   */
+  name: string;
+  /**
+   * OrgId is the ID of an organisation the team belongs to.
+   */
+  orgId: number;
+  /**
+   * TODO - it seems it's a team_member.permission, unlikely it should belong to the team kind
+   */
+  permission: TeamPermissionLevel;
```

Comment:
Look into this one I can see that in backend Team struct don't include this one but [TeamDTO](https://github.com/grafana/grafana/blob/main/pkg/services/team/model.go#L91-L101) do. The names was reversed in front end ^^. 

Do we need this field, and if we do we should probably make it optional

Reply:
I don't know if it's needed, as all the other types it was just copied over from grafana/public/src/types